client/src/components/common/boost/BoostCalibration.tsx

```diff
@@ -46,13 +46,6 @@ export default function BoostCalibration({
     [calibValue, onSet, setValidated],
   );
 
-  const handleKeyPressed = useCallback(
-    (event: React.KeyboardEvent<HTMLInputElement>) => {
-      if (event.key === 'Enter') handleSubmit(event);
```

Comment:
@kdav108 The type error was here because `event.currentTarget: HTMLInputElement` is not convertable to `HTMLFormElement`.

Reply:
Nah I got that, I wanted to know where that error was actually displayed? It wasn't showing up in terminal when you run the application nor in the static checking in VS code, so where was it that you picked it up, or was this just based on your knowledge?

Reply:
Oh, for me it was picked up from VSCode as well as when I ran `yarn start` or `yarn run storybook`.

Reply:
![image](https://user-images.githubusercontent.com/50545432/113479427-d0b16480-94da-11eb-805b-91a0ffc85176.png)